content/departments/technical-success/support/process/Support Agent and Customer Slack Channels

```diff
@@ -0,0 +1,13 @@
+##Support Agent and Customer Slack Channels 
+
+To better support our customers we use a Support Agent that will need to be integrated into customer slack channels. This will allow us to receive customer request quickly and directly triage the request from Slack. To ensure that the Support Agent is enabled there are a few steps that need to be completed. 
+
+**Step 1** Once the customer slack channel has been created, please @ menttion CS. This will add the support engineering team to the channel, triggering CS leadership to integrate the Support Agent. 
+
+**Step 2** Once the integration is completed the customer will be able to submit request using the box labled <Create Request>
+
+**Step 3** To communicate the how to reach support AE's TA's and CE's can post the following in the customer channel: Hi team, we wanted to share the best way to reach our support team if you run into any questions. There will be a message and a button at the bottom of the channel that reads “Question for Support? Create Request.” The Create Request button will open a form where you can fill out information for a Question or Issue. Let us know if you have any questions, Thank you!”
```

Comment:
From: **Step 3** To communicate the how to reach support AE's TA's and CE's can post the following in the customer channel:

To: **Step 3** Send a message in the channel explaining how the customer can utilize the Support Agent: